tests/components/withings/common.py

```diff
@@ -233,11 +230,9 @@ async def setup_profile(self, user_id: int) -> ConfigEntryWithingsApi:
         result = await self._hass.config_entries.flow.async_configure(result["flow_id"])
         assert result.get("type") == "form"
         assert result.get("step_id") == "profile"
-        assert result.get("data_schema").schema["profile"].container == [
-            profile.profile for profile in self._profile_configs
-        ]
+        assert "profile" in result.get("data_schema").schema
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5737505)